charset_normalizer/md.py

```diff
@@ -495,18 +461,13 @@ def is_suspiciously_successive_range(
             return False
 
     # Japanese Exception
-    if unicode_range_a in ["Katakana", "Hiragana"] and unicode_range_b in [
-        "Katakana",
-        "Hiragana",
-    ]:
-        return False
-
-    if unicode_range_a in ["Katakana", "Hiragana"] or unicode_range_b in [
-        "Katakana",
-        "Hiragana",
-    ]:
+    range_a_jp_chars, range_b_jp_chars = unicode_range_a in ("Hiragana", "Katakana"), \
+                                         unicode_range_b in ("Hiragana", "Katakana")
```

Comment:
Tuple unpacking is likely to be slower than two separate assignment statements.

Reply:
I don't agree, it is not expected to be slower, and I see no difference on my machine, neither for the separate assignments nor for the test conditions.
The current solution is more readable I think, what do you say @Ousret ?

Reply:
I am in favour of @adbar for this. The core Python offer pretty high speed regarding tuple unpacking. Using timeit, I could see a difference of 2ns in favour of multiple assignments but it is too small to consider switching the writing.